.github/workflows/deploy-storybook.yml

```
@@ -2,34 +2,35 @@ on:
 name: 🚀 Deploy Storybook
 
 jobs:
-  deploy_sb:
+  deploy-chromatic:
     runs-on: ubuntu-latest
-    name: 🚀 Deploy Storybook
+    name: 🚀 Deploy Chromatic
     steps:
       - name: Checkout code
-        uses: actions/checkout@v4
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)